indico/modules/events/registration/schemas.py

```diff
@@ -33,3 +39,86 @@ def sort_list(self, data, many, **kwargs):
         if many:
             data = sorted(data, key=lambda tag: natural_sort_key(tag['title']))
         return data
+
+
+# Schemas for the Check-in app API
```

Comment:
Pretty clear from the name IMHO ;) And I'm sure sooner or later someone will add new (unrelated) classes at the bottom of the file anyway